it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -37,17 +37,17 @@
 #
 # The following are used by the compose back end
 #
-## @name NS_MSG_UNABLE_TO_OPEN_FILE 
+## @name NS_MSG_UNABLE_TO_OPEN_FILE
 12500=Impossibile aprire il file %P0%.
 
 ## @name NS_MSG_UNABLE_TO_OPEN_TMP_FILE
-12501=Impossibile aprire il file temporaneo %P0%. Controllare l'impostazione 'Cartella temporanea' e riprovare.
+12501=Impossibile aprire il file temporaneo %P0%. Controlla le tue impostazioni 'Cartella temporanea'.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12501=Impossibile aprire il file temporaneo %P0%. Controlla le tue impostazioni 'Cartella temporanea' e riprova.